### PR TITLE
feat : display locations and quantities in stock party table, handle out-of-stock items

### PR DESCRIPTION
**Pull request by charity553**
[…ntities.](https://www.loom.com/share/0ec7a3e9fb5e4c518e5af220c89ee4e0?sid=b9d9b90d-bf87-4531-b009-7cd62e2d9d47) Added handling for out of stock items.

## Requirements

- [ ] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Screenshots
https://www.loom.com/share/0ec7a3e9fb5e4c518e5af220c89ee4e0?sid=67dcdf22-6d56-4ada-95fd-2bb34768159b

*None.*

## Related Issue

*None.*

## Other

*None.*

### CODE REVIEW

**Comment by makombe** [comment]
@charity553 Please fix the conflicts. Thanks

**Comment by charity553** [comment]
on it @makombe

File: packages/esm-billing-app/src/billable-services/billiable-item/drug-order/drug-order.component.tsx [hunk 1]
```diff
@@ -25,21 +25,29 @@ const DrugOrder: React.FC<DrugOrderProps> = ({ order }) => {
...
+              <span>{item.partyName}</span>
+              <span>
+                {' '}
+                {Math.round(item.quantity)} {item.quantityUoM}(s){' '}
+              </span>
+            </div>
+          ))}
+        </>
+      ) : (
+        <div className={styles.red}>{'Drug Is Not Available  / Out of Stock'}</div>
```
Comment (donaldkibet) [inline packages/esm-billing-app/src/billable-services/billiable-item/drug-order/drug-order.component.tsx:48]: We should use https://react.carbondesignsystem.com/?path=/docs/components-notifications-inline--overview for this

**Review by donaldkibet — APPROVED** [review]
LGTM left a suggestion. 🤝

**Pull request merged**